Source/SA/RFmx SpecAn/RFmxSpecAn.cs

```diff
@@ -44,33 +40,313 @@ public struct AutoLevelConfiguration
+                    Rbw_Hz = 20e6,
+                    RbwFilterType= RFmxSpecAnMXTxpRbwFilterType.None,
+                    RrfAlpha=0.01,
+
+                    AveragingEnabled=RFmxSpecAnMXTxpAveragingEnabled.False,
+                    AveragingCount = 10 ,
+                    AveragingType = RFmxSpecAnMXTxpAveragingType.Rms,
+                 };
+            }
+        }
+
+        public struct AcpCommonConfiguration
+        {
+            public RFmxSpecAnMXAcpPowerUnits PowerUnits;
+            public RFmxSpecAnMXAcpAveragingEnabled AveragingEnabled;
+            public int AveragingCount;
+            public RFmxSpecAnMXAcpAveragingType AveragingType;
+            public RFmxSpecAnMXAcpFftWindow FftWindow;
+            public double FftPadding;
+
+            public RFmxSpecAnMXAcpRbwAutoBandwidth RbwAuto;
+            public double Rbw_Hz;
+            public RFmxSpecAnMXAcpRbwFilterType RbwFilterType;
+
+            public RFmxSpecAnMXAcpSweepTimeAuto SweepTimeAuto;
+            public double SweepTimeInterval_s;
+            public static AcpCommonConfiguration GetDefault()
+            {
+                return new AcpCommonConfiguration
+                {
+                    PowerUnits = RFmxSpecAnMXAcpPowerUnits.dBm,
+                    AveragingEnabled = RFmxSpecAnMXAcpAveragingEnabled.False,
+                    AveragingCount = 10,
+                    AveragingType = RFmxSpecAnMXAcpAveragingType.Rms,
+                    FftWindow = RFmxSpecAnMXAcpFftWindow.FlatTop,
+                    FftPadding = 1,
+                    RbwAuto = RFmxSpecAnMXAcpRbwAutoBandwidth.True,
+                    Rbw_Hz = 10e3,
+                    RbwFilterType = RFmxSpecAnMXAcpRbwFilterType.Gaussian,
+                    SweepTimeAuto = RFmxSpecAnMXAcpSweepTimeAuto.True,
+                    SweepTimeInterval_s = 1e-3
+
+                    };
+            }
+        }
+        public struct AcpCarrierChannelConfiguration
+        {
+            public double IntegrationBandwidth_Hz;
+            public RFmxSpecAnMXAcpCarrierRrcFilterEnabled RrcFilterEnabled;
+            public double RrcAlpha;
+            public RFmxSpecAnMXAcpCarrierMode Mode;
+            public double Frequency;
+            public static AcpCarrierChannelConfiguration GetDefault()
+            {
+                return new AcpCarrierChannelConfiguration
+                {
+                    IntegrationBandwidth_Hz = 18e6,
+                    RrcFilterEnabled = RFmxSpecAnMXAcpCarrierRrcFilterEnabled.False,
+                    RrcAlpha = 0.22,
+                    Mode = RFmxSpecAnMXAcpCarrierMode.Active,
+                    Frequency = 0.00
+                };
+        
+            }
+        }
+        public struct AcpOffsetChannelConfiguration
+        {
+            public double IntegrationBandwidth_Hz;
+            public RFmxSpecAnMXAcpOffsetEnabled OffsetEnabled;
+            public double OffsetFrequency_Hz;

```

Comment:
```suggestion
            public double Frequency_Hz;
```

That or we need a similar naming for carriers